Dockerfile

```diff
@@ -20,12 +21,15 @@ RUN apk add --update \
         openssl-dev \
         unixodbc-dev \
         && \
+    pip list --outdated --format=freeze | grep -v '^\-e' | cut -d = -f 1  | xargs -n1 pip install --no-cache-dir -U \
+        && \
     pip install --upgrade \
         pip \
         setuptools \
-        virtualenv \
+        virtualenv && \
+    pip3 list --outdated --format=freeze | grep -v '^\-e' | cut -d = -f 1  | xargs -n1 pip3 install --no-cache-dir -U \
```

Comment:
Should we move this below the pip upgrade on the next line, so these updates happen with the updated pip?

Reply:
Yeah that's a good idea. I'll push an update.